local_sales/db/seeds/production.rb

```diff
@@ -0,0 +1,24 @@
+require 'faker'
```

Comment:
Why do you need yo seed in production?

I'd not recommend you to do it, please check this [article](https://ninjadevel.com/seeding-database-ruby-on-rails/) and let me know your thoughts.

Reply:
Because we need to have 2 kind of users in production,we have decided to have only a super administrator,so he must be created,but considering the article approach we can delete the instructions that uses faker and it just keep the users creation in the production seeder.

Reply:
Indeed, remember that in production, the values ​​that you are going to have must be real.